src/app/builder/builder.component.html

```diff
@@ -25,6 +25,16 @@
       width="600"
       [src]="previewIframeSrc"></iframe>
   </div>
+  <div *ngIf="playPlaylist" id="preview-playlist-wrap">
```

Comment:
Sometimes it does and sometimes it doesn't. Playlist wasn't showing at all when I first used this and assumed I had to check play latest then select playlist. I checked the iframe shown with preview and it did not have the sp param.